scripts/install_mt4.verb

```diff
@@ -22,12 +22,10 @@ load_install_mt4()
     w_try_cd "$W_CACHE/$W_PACKAGE"
     w_ahk_do "
         Run, "$(find . -name "*.exe" -print -quit)"
-        WinWait, 4 Setup
-        ControlClick, Button1
-        Sleep 100
-        ControlClick, Button3
-        WinWait, 4 Setup, Installation successfully completed
-        ControlClick, Button4
+        WinWait, XM UK MT4 Setup
```

Comment:
Is it possible to use the wildcard like `*4 Setup` in case setup from a different broker is used?